myuw/dao/student.py

```diff
@@ -0,0 +1,129 @@
+"""
+This module will provide a
+"""
+from uw_sws.enrollment import enrollment_search_by_regid
+from uw_sws.term import get_current_term
+
+
+def get_student_status(regid):
+    """
+    This will return a dictionary with the student status over time alongside
+    various rollup variables
+    :param regid:
+    :return: dict
+    """
+    enrollments = enrollment_search_by_regid(regid)
+
+    return {
+        'majors': _get_majors(enrollments),
+        'class_level': _get_class_standings(enrollments)
+    }
+
+
+def get_majors(regid):
+    """
+    Returns a dictionary with a dict of majors by terms in the 'majors'
+    attribute, and a 'current' and 'rollup' major field
+    :param regid
+    :return: dict
+    """
+    enrollments = enrollment_search_by_regid(regid)
+
+    return _get_majors(enrollments)
+
+
+def get_rollup_and_future_majors(majors):
```

Comment:
Would it be OK to change "rollup" to "last" (to be more specific)?

Reply:
The reason rollup is called rollup is because we default to current, then go to future, then go to last